third_party/2and3/werkzeug/serving.pyi

```diff
@@ -2,12 +2,20 @@ import sys
 from typing import Any, Optional
 
 if sys.version_info < (3,):
-    from SocketServer import ThreadingMixIn, ForkingMixIn
+    from SocketServer import ThreadingMixIn
     from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
 else:
-    from socketserver import ThreadingMixIn, ForkingMixIn
+    from socketserver import ThreadingMixIn
     from http.server import HTTPServer, BaseHTTPRequestHandler
 
+if sys.platform == "win32":
+    class _ForkingMixIn: ...
```

Comment:
```suggestion
    class _ForkingMixIn(object): ...
```

(Python 2 compatibility)

Also, it makes sense to actually name this `ForkingMixIn` (and re-export if platform != win32), since werkzeug defines it under this name if it doesn't exist.